packages/react/src/icon/docs/overview.mdx

```diff
@@ -1,14 +1,68 @@
+Meaningful icons can be used to enhance the experience for sighted users; making it easier to scan content, identify actions and understand user interfaces. The icon component gives you access to a suite of system icons that are designed to represent common states and universal actions.
+
+<DoHeading />
+
+- Use icons, that all users understand, and that clearly represent an intended meaning or context.
+- Use icons to support visual recognition of repeatable or consistent system wide states or actions
+- Always use a meaningful text label and accessible name when using an icon in an action
+- Avoid using bespoke icons in your applications as you may confuse users
+- Only consider using an icon without a visible label if there is insufficient space and including the label will detract from the user experience.
+
+<DontHeading />
+
+- Use icons that are unconventional or are not commonly understood to represent the action or state - this will confuse users.
+
+## Icons
+
+Each icon in our library is simple but detailed enough to ensure users can understand them. Icons should not be too detailed, so they are clear and legible at all available sizes. [See the full list of available icons.](/foundations/icons)
 
 > Do you have a user need for a new or missing icon? [Get in touch](/about) to discuss an icon contribution.
+
+## Icons and labels
+
+Provide a meaningful label and accessible name with the icon to help all users quickly recognise and understand what action will occur. Icons are small visual elements that can be interpreted differently by people. Using icons without labels can confuse some users - even the simplest icons will be new or uncommon to some users.
+
+An icon can be used without a visible label if there is insufficient space and the context provides clear meaning to the icon. An example of this is a mobile search bar. Including a label for the action will take up essential space needed by the search input, and the context provides clear meaning to the icon.
+
+Use icons and labels together that are commonly used and accurately communicate an action, context or outcome. For example, a flag [Toggle button](/components/toggle-button) can be used without a visible label provided there is enough context surrounding the icon, such as in a mailbox application to action its importance or required attention. [Toggle button](/components/toggle-button) will provide an accessible label for assistive technologies.
+
+## Icon Position
+
+When using an icon with a button or link, the icon should appear before the text except when the icon indicates direction, e.g. Next or external links.
+
+## Sizes
+
+Icons are available in sizes medium, large and extra large.
+
+Medium icons are used in small buttons, in components like accordions, drop-down menus, breadcrumbs and links that are nested in content paragraphs. Always consider the complexity of an icon before using the small variant, as small complex icons may be hard to read for some users.
```

Comment:
This feels weird to tack a thing about small on the end. We should lead with small (so the content increases in size order), states what small is for, but also warns.